Readme.md

```diff
@@ -0,0 +1,6 @@
+The official android app for MusicBrainz. MusicBrainz for Android lets users search metadata all of the the metadata available on MusicBrainz. Users can scan barcode to search albums. Registered users can view, edit and modify their collections. Users can also tag their audio files.
```

Comment:
Maybe stating that the project is part of GSoC is useful, as stated [here](https://developers.google.com/open-source/gsoc/help/work-product):

> Make sure the description clearly notes that this is for Google Summer of Code.

```suggestion
# MusicBrainz Android app

This project was part of  Google Summer of Code 2019.

The official android app for MusicBrainz. MusicBrainz for Android lets users search metadata all of the the metadata available on MusicBrainz. Users can scan barcode to search albums. Registered users can view, edit and modify their collections. Users can also tag their audio files.
```